src/project.js

```diff
@@ -70,31 +70,51 @@ function projectionReducer(location) {
     newLocation.forestType = forestType;
   }
 
-  newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
+  if (location.altitudinalZone && targetaltitudinalZone !== undefined) {
+    newLocation.altitudinalZone = getNextHeigtLevel(location.altitudinalZone);
```

Comment:
Can you rename the function to getNextAltitudinalZone. BTW: adding a comment to the function about why next zone is lower might be a good idea :)

Reply:
Agree!